README.md

```diff
@@ -1,2 +1,172 @@
+# 📖 Hello Microverse <a name="about-project"></a>
+
+> Hello Microverse is a simple project using html and css.
+
+## 🛠 Built With <a name="built-with"></a>
+1. HTML5
+2. CSS
+3. HTML-CSS linters
+### Tech Stack <a name="tech-stack"></a>
+
+> 
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+### Key Features <a name="key-features"></a>
+1. Added index.html file
+2. Added stle.css file
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+Comming soon
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+1. Code editor
+2. Web browser
+3. Git
+
+### Setup
+
+Clone this repository to your desired folder:
+Run the below command:
+
+```sh
+cd <your disered folder name >
+git clone git@github.com:HossainAraf/Hello.git
+```
+### Install
+
+Install with below command:
+
+```sh
+  cd <your disered folder name >
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following:
+
+Open index.html file either directly or using live server 
+
+### Run tests
+
+Comming soon
+
+### Deployment
+
+Comming soon
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 Md Arafat Hossain
+
+- GitHub: HossainAraf (https://github.com/HossainAraf)
+- LinkedIn: Md. Arafat Hossain (https://linkedin.com/in/md-arafat-hossain-111403275)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] Add header
+- [ ] Add footer
+- [ ] Add javascript file
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give a ⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse team
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Kindly add a LICENSE to your project's files since the link is not working.

---

- _Please refer to this simple [tutorial](https://www.youtube.com/watch?v=8Dm7XXvtqXI) which will help you create one, please do not forget to choose the "MIT" license._